src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -34,16 +34,18 @@ public class LibraryServices : LogSourceBase, IDisposable
 
```

Comment:
`LibraryServices` is keeping a reference to `IPathManager` to help with resolving library paths.